<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>432</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>432</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>432</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>红外成像软件</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    border: 2px solid gray;
    border-radius: 3px;
	margin-top: 1ex;
	padding-bottom: 4px;
	padding-top: 7px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    /*subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #rgb(244, 244, 244), stop: 1 #rgb(244, 244, 244));
}

QGroupBox#groupBox_6 {   /*帧率与温度组设置无边框*/
	border: none;
	margin-top: 0;
	padding-top:0px;
}

QGroupBox#groupBox_6 {
	padding: 0 0;

}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>432</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>384</width>
        <height>350</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>384</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="m_startButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>340</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>打 开</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="m_stopButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>340</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>关 闭</string>
       </property>
      </widget>
      <widget class="QFrame" name="m_showFrame">
       <property name="geometry">
        <rect>
         <x>25</x>
         <y>35</y>
         <width>324</width>
         <height>256</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>256</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>324</width>
         <height>256</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(130, 130, 130)</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="m_showLabel">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>324</width>
            <height>256</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;显示&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton_begin_video">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>开始录制</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>171</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>true</underline>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="toolTipDuration">
        <number>-1</number>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: rgb(240, 240, 240);</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="3" column="1">
         <widget class="QRadioButton" name="radioButton_color">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>热金属</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QRadioButton" name="radioButton_gray">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="text">
           <string>黑白</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QRadioButton" name="radioButton_rainbow">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="text">
           <string>彩虹</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QRadioButton" name="radioButton_gray_inverse">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>白黑</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="m_imageTakingButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>拍  照</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_pathChoosingButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>选择存储路径</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_pauseButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>暂  停</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_finish_video">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>停止录制</string>
       </property>
      </widget>
      <widget class="QPushButton" name="m_open_file">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>幼圆</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>打开图片/视频</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
